### ISSUE TITLE
Cura 15.04.6 Mac Line Endings Error

### ISSUE THREAD

**Issue by Omniwheel** [daid/LegacyCura#1491]
On Mac when editing start and end GCode in Cura, Cura inserts Windows carriage returns. When Cura is closed and then reopened the Windows carriage returns turn into Unix new lines.

This is particularly problematic on some printers like XYZ machines. Due to a separate bug, XYZWare on Mac and/or XYZ printers don't properly handle Windows carriage returns.

**Comment by BagelOrb** [comment]
Does the problem also occur in cura 2.1?
Op 3 jun. 2016 19:53 schreef "Karthik Chokkaram" notifications@github.com:

**Comment by Omniwheel** [comment]
No... I don't think there is a way to edit start or end G-code in 2.1.

**Comment by BagelOrb** [comment]
Ah of course.
Op 4 jun. 2016 19:19 schreef "Karthik Chokkaram" notifications@github.com:
